classlib/src/main/java/org/teavm/classlib/java/util/TArrayDeque.java

```diff
@@ -38,8 +39,8 @@ public TArrayDeque(TCollection<? extends E> c) {
         } else {
             array = new Object[c.size() + 1];
             int index = 0;
-            for (TIterator<? extends E> iter = c.iterator(); iter.hasNext();) {
-                array[index++] = iter.next();
+            for (TIterator<? extends E> it = c.iterator(); it.hasNext(); ) {
```

Comment:
What is the purpose of this change? BTW, I'm not sure if checkstyle passes extra space after last semicolon.

Reply:
BTW, this place is a good candidate to use `var`

Reply:
Some "strange" changes appeared automatically in final preparation to PR. I'm just resolving IDEA issues by reformatting and selecting "obvious" fix for warning by pressing Alt-Enter.
I suppose that IDEA imported your codestyle for formatting and checkstyle will pass this change.
At least `./gradlew publishToMavenLocal` works.